lib/quicly.c

```diff
@@ -4621,6 +4636,9 @@ const char *quicly_event_attribute_names[] = {NULL,
                                               "cc-exit-recovery",
                                               "acked-packets",
                                               "acked-bytes",
+                                              "min_rtt",
+                                              "smoothed_rtt",
+                                              "latest_rtt",
```

Comment:
Can we use `-` instead of `_`? I think that is the convention we have so far.

Other than that the PR looks good to me.

Reply:
Done.